### PR TITLE
feat: add wallet disclaimer

### LINKED ISSUE #8283: [Feature]: CLI Wallet Legal Disclaimer
## Story

As a CLI Wallet team
We want our users to fully understand what the app allows them to do and the risks involved   
So that all users can make an informed decision on whether or not they wish to accept the risks and use the app

## Acceptance Criteria

When I open the CLI Wallet for the first time...
- [ ] I am provided with brief legal disclaimer describing what the app does and the risks involved 
- [ ] I can acknowledge the disclaimer and continue to the CLI wallet
- [ ] I can not continue to use the CLI wallet without acknowledging the disclaimer
- [ ] I am provided with relevant information based on the network I am connected to (i.e. different message for mainnet v testnet) 

## Tasks

- [ ] Implement
- [ ] Write unit tests
- [ ] Write end to end tests
- [ ] Code review
- [ ] Verify acceptance criteria met
- [ ] Exploratory QA testing (capturing bugs and new edge cases in this or new tickets) 
- [ ] Write automated regression tests for the A/Cs (expanding on A/Cs where possible) 
- [ ] Release

## Notes

[Mainnet content has been drafted by legal here](https://docs.google.com/document/d/1bg7dKCbV-HRgf8Jof7W_guD1JcO0nbl36hSVJ0CObUQ/edit)

### PR DESCRIPTION
**Pull request by jeremyletang**
close #8283

### CODE REVIEW

File: cmd/vegawallet/commands/disclaimer.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+omissions by users in connection with or as a result of their Vega Wallet or other
+associated wallet being compromised.
+`
+
+var (
+	disclaimerLong = cli.LongDesc(`
+		Creates the folders, the configuration files and RSA keys needed by the service
+		to operate.
+	`)
+)
```
Comment (ValentinTrinque) [inline cmd/vegawallet/commands/disclaimer.go:46]: Wrong description

**Review by ValentinTrinque — CHANGES_REQUESTED** [review]

**Comment by wwestgarth** [comment]
I'm assuming this is for https://github.com/vegaprotocol/vega/issues/8283? You've got a link to this PR in the `#closes` bit in the PR description, which is a bit too recursive for my liking :p

**Comment by jeremyletang** [comment]
> I'm assuming this is for #8283? You've got a link to this PR in the `#closes` bit in the PR description, which is a bit too recursive for my liking :p

good point.

**Review by ValentinTrinque — APPROVED** [review]
